account_brand/models/account_move.py

```diff
@@ -13,7 +13,13 @@ class AccountMove(models.Model):
 
     def _is_brand_required(self):
         self.ensure_one()
-        if self.move_type in ("in_invoice", "in_refund"):
+        if self.move_type in (
+            "in_invoice",
+            "in_refund",
+            "entry",
+            "out_receipt",
+            "in_receipt",
+        ):
```

Comment:
Explicit is better than implicit

```suggestion
        if self.move_type not in ("out_invoice", "out_refund"):
```